### PR TITLE
update call to sponsors section for boulder

### PR DESCRIPTION
**Pull request by stew**

### CODE REVIEW

File: _events/2016-05-summit-boulder.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ poster_hero: "/img/media/boulder.jpg"
 
 location_section: true
 papers_section: true
+
+featured: true
```
Comment (larsrh) [inline _events/2016-05-summit-boulder.md]: Have you checked what the front page looks like with the `featured` flag enabled? There's no thumbnail yet, so I suspect it looks broken.

Comment (stew) [inline _events/2016-05-summit-boulder.md]: hah, no, i was stabbing randomly at that one, don't know what it means. I'll take it out

File: _events/2016-05-summit-oslo.md [hunk 2]
```diff
@@ -14,7 +14,6 @@ location_section: true
 papers_section: true
 sponsors_section: true
 
-featured: true
```
Comment (stew) [inline _events/2016-05-summit-oslo.md]: i suck

please hold

**Comment by stew** [comment]
crap what have I done?

**Comment by larsrh** [comment]
It looks like you merged, but the commit is not there. Did you force push?

**Comment by stew** [comment]
I meant to update this pr, pushed to origin by mistake, then immediately force pushed it back

**Comment by stew** [comment]
so sorry

**Comment by larsrh** [comment]
No problem, that can happen.

**Comment by stew** [comment]
opened again as #88

**Pull request merged**